api/src/migrations/1619716338943-CreateUser.ts

```diff
@@ -0,0 +1,29 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class CreateUser1619716338943 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`
+      CREATE TABLE  IF NOT EXISTS "user" (
+        id uuid NOT NULL DEFAULT uuid_generate_v4(),
+        kc_id character varying  NOT NULL,
+        first_name character varying,
```

Comment:
do we need to store first_name and last_name?

Reply:
I don't believe we do. Only the first name is in the mock ups.

Reply:
ok I'm totally wrong, first and last name are in the mock ups!